### DISCUSSION TITLE
Edit on multiple cursors not working?

### DISCUSSION THREAD

**Discussion by lkdm** [vscode-neovim/vscode-neovim#1486] [Q&A]
Sorry I'm a Nvim noob, and I'm trying to figure out how to edit at multiple cursors.

Here's the process I've tried:

```
CTRL + v	Visual mode
jjjjj		Select 5 rows
mi		Spawn multiple cursors
Shift + i	Insert
```

This just inserts on the most recent cursor location, which is not what I want.

**Comment by xiyaowong** [comment]
Need a screen record.
